src/assets/scripts/client/scope/RadarTargetModel.js

```diff
@@ -0,0 +1,411 @@
+    }
+
+    /**
+     * Enable handlers
+     *
+     * @for RadarTargetModel
+     * @method disable
+     */
+    disable() {
+        this._eventBus.off(EVENT.SET_THEME, this._setTheme);
+    }
+
+    /**
+     * Disable handlers
+     *
+     * @for RadarTargetModel
+     * @method enable
+     */
+    enable() {
+        this._eventBus.on(EVENT.SET_THEME, this._setTheme);
+    }
+
+    /**
+    * Mark this radar target as being controlled by "our" ScopeModel
+    * Note that this will eventually be reworked so we can specify which
+    * scope has control, not just whether or not "we" do.
+    *
+    * @for RadarTargetModel
+    * @method markAsNotOurControl
+    */
+    markAsNotOurControl() {
+        this._isUnderOurControl = false;
+    }
+
+    /**
+     * Mark this radar target as being controlled by "our" ScopeModel
+     * Note that this will eventually be reworked so we can specify which
+     * scope has control, not just whether or not "we" do.
+     *
+     * @for RadarTargetModel
+     * @method markAsOurControl
+     */
+    markAsOurControl() {
+        this._isUnderOurControl = true;
+    }
+
+    /**
+     * Change the direction and/or length of the data block leader line
+     *
+     * @for RadarTargetModel
+     * @method moveDataBlock
+     * @param commandArguments {string}
+     * @return {array} [success of operation, system's response]
+     */
+    moveDataBlock(commandArguments) {
+        if (_isEmpty(commandArguments)) {
+            return [false, 'ERR: BAD SYNTAX'];
+        }
+
+        let desiredDirection = commandArguments;
+        let desiredLength = '';
+
+        // FIXME: Extract this to a constants file!
+        if (commandArguments.indexOf('/') !== INVALID_NUMBER) {
+            const argumentPieces = commandArguments.split('/');
+            desiredDirection = parseInt(argumentPieces[0], DECIMAL_RADIX);
+            desiredLength = parseInt(argumentPieces[1], DECIMAL_RADIX);
+
+            if (_isEmpty(argumentPieces[0])) {
+                desiredDirection = '';
+            }
+        }
+
+        if (desiredLength > 6 || desiredLength < 0) {
+            return [false, 'ERR: LEADER LENGTH 0-6 ONLY'];
+        }
+
+        if (desiredDirection !== '' && !_isNaN(desiredDirection)) {
+            const dataBlockPositionMap = { 8: 360, 9: 45, 6: 90, 3: 135, 2: 180, 1: 225, 4: 270, 7: 315, 5: 'ctr' };
+
+            if (!_has(dataBlockPositionMap, desiredDirection)) {
+                return [false, 'ERR: BAD SYNTAX'];
+            }
+
+            this._dataBlockLeaderDirection = dataBlockPositionMap[desiredDirection];
+        }
+
+        if (desiredLength !== '' && !_isNaN(desiredLength)) {
+            this._dataBlockLeaderLength = desiredLength;
+        }
+
+        return [true, 'ADJUST DATA BLOCK'];
+    }
+
+    /**
+     * Reset all properties to their default state
+     *
+     * @for RadarTargetModel
+     * @method reset
+     */
+    reset() {
```

Comment:
move to after `_init()` with other lifecycle methods defined in calling order

Reply:
done